mongosql/src/mir/optimizer/match_null_filtering/mod.rs

```diff
@@ -184,8 +184,6 @@ impl Visitor for NullableSetter {
         node
     }
 
-    // todo: only set nullable for expressions that don't include a null literal
```

Comment:
drive-by -- forgot to remove this from the previous pr that worked on this optimization 🚗